.github/workflows/api_tests.yaml

```diff
@@ -11,19 +11,17 @@ jobs:
         name: curl tests
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v4
             - uses: shivammathur/setup-php@v2
               with:
                   # test the lowest version, to make sure checks pass on it
                   php-version: ${{ matrix.php-version }}
                   extensions: json, mbstring, pdo, curl, pdo_sqlite
                   coverage: none
-            - name: Initialise
-              run: |
-                  sudo composer self-update -q	
-            - name: Install dependencies
-              run: |
-                  sudo COMPOSER_MEMORY_LIMIT=-1 COMPOSER_PROCESS_TIMEOUT=60 composer update --prefer-dist --no-progress
+            - name: Install composer dependencies
+              uses: php-actions/composer@v6
```

Comment:
I prefer write the composer install command but install `composer` and Symfony cli from `shivammathur/setup-php@v2`

See: https://github.com/bolt/core/pull/3533

Reply:
Great suggestion, I was not aware `shivammathur/setup-php@v2` also installs composer (normally I work with Gitlab). Lets see whether the new commit still works.

Reply:
You need add `tools: composer` after `coverage: none` to install on the `shivammathur/setup-php@v2` to install the latest composer version (like https://github.com/bolt/core/pull/3533).

Reply:
According to the action documentation "The latest stable version of composer is set up by default."

Reply:
Ho nice !